install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -7,7 +7,12 @@ metadata:
   name: cilium-operator
   namespace: {{ .Release.Namespace }}
 spec:
+  # We support HA mode only for Kubernetes version > 1.14
```

Comment:
It'd be nice to a have a reference to give more context here as to why. Something like "See \<file or something\> for more details."